### PR TITLE
Extracted aws sqs credentials and client into a different class and using that for publishing/subscribing

### PR DESCRIPTION
**Pull request by rocky02**
This PR deals with extracting the code for AWS SQS credentials and creation of SQS Client from Publisher and Subscriber into a single aws_sqs_client class which can be used by both.

Addressing Issue #7

### CODE REVIEW

File: aws_sqs.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class AwsSQS
+
+  attr_reader :client
+
+  def initialize 
+    creds = Aws::Credentials.new(AWS["access_key_id"], AWS["secret_access_key"])
+    @client = Aws::SQS::Client.new(region: AWS["region"], credentials: creds)
+  end
+
+end
```
Comment (glen) [inline aws_sqs.rb]: Newline at end of the file.

Comment (rocky02) [reply aws_sqs.rb]: done.

File: aws_sqs.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class AwsSQS
```
Comment (glen) [inline aws_sqs.rb]: Would be more explicit if we name the class `AwsSQSClient`  or something like that.

Comment (rocky02) [reply aws_sqs.rb]: I renamed it.. will push after all changes are done.

**Pull request merged**